dpnp/dpnp_algo/dpnp_algo.pxd

```diff
@@ -299,8 +299,6 @@ cdef extern from "dpnp_iface_fptr.hpp" namespace "DPNPFuncName":  # need this na
         DPNP_FN_SINH_EXT
         DPNP_FN_SORT
         DPNP_FN_SORT_EXT
-        DPNP_FN_SQRT
-        DPNP_FN_SQRT_EXT
```

Comment:
remove `DPNP_FN_SQRT_EXT` from `dpnp/backend/include/dpnp_iface_fptr.hpp` as well and entries from `dpnp/backend/kernels/dpnp_krnl_elemwise.cpp`.